index.js

```diff
@@ -69,6 +69,13 @@ import EventEmitter from 'wolfy87-eventemitter'
           window.open(protocol + '://'+ instance.proxy_host + '-' + port + '.direct.' + self.opts.baseUrl.split('/')[2] + this.attributes.href.value, '_blank');
         }
       };
+      actions[n].oncontextmenu = function(evt) {
```

Comment:
Seems like this code is pretty similar to the one in `onclick`. Can we do something to avoid duplication?